### ISSUE TITLE
MappedFile::actualSize might get caught in a loop

### ISSUE THREAD

**Issue by minborg** [OpenHFT/Chronicle-Bytes#115]
It is unclear when `FileChannel::size` will ever throw an `ArrayIndexOutOfBoundsException` but if it does, the method `MappedFile::actualSize` might get caught in an infinite loop until the stack overflows.

```
    public long actualSize() throws IORuntimeException {
        boolean interrupted = Thread.interrupted();
        try {
            return fileChannel.size();

        } catch (ArrayIndexOutOfBoundsException aiooe) {
            // try again.
            return actualSize();


```

**Comment by peter-lawrey** [comment]
Added a comment, that I saw this once coming out of the JVM.
I should have recorded it but I will leave it for now.

**Comment by hft-team-city** [comment]
Released in [Chronicle-Bytes-2.20.43](https://github.com/OpenHFT/Chronicle-Bytes/releases/chronicle-bytes-2.20.43), [BOM-2.20.67](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.67)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Bytes-2.20.101](https://github.com/OpenHFT/Chronicle-Bytes/releases/chronicle-bytes-2.20.101), [BOM-2.20.134](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.134)
